Add the 'Getting Started with TabPanel' tutorial (#2200)
DevExpress/devextreme-documentation: concepts/05 UI Components/TabPanel/01 Getting Started with TabPanel/50 Navigate Between Tabs.md

Specify the following properties to configure navigation between tabs:      
- [swipeEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#swipeEnabled)      

> Configures swiping. This property's default value is **true** on touch devices and **false** on others.

Reviewer: > swiping

-ing

>This property's default value is **true** on touch devices and **false** on others.

Default values are listed in the API reference.

```suggestion
Specifies whether users can switch between views with the swipe gesture.
```

- [loop](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#loop)      

> Specifies whether to loop items.

Reviewer: ```suggestion
Specifies whether to loop views.
```

- [animationEnabled](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/#animationEnabled)      

> Specifies whether to animate the change of the selected item.

Reviewer: ```suggestion
Specifies whether to animate the change of the current view.
```

                        formData: employeeData,
                        items: ["name", "position", "hireDate", "officeNumber"]
                    }).dxForm('instance');
                }
            }, {
                title: "Notes",
                icon: "comment",
                template: function (itemData, itemIndex, element) {
                    let textAreaDiv = $("<div style='padding:15px; height: 100%'>")
                    textAreaDiv.appendTo(element);
                    textAreaDiv.dxTextArea({
                        value: employeeData.notes
                    }).dxTextArea('instance');
                }
            }, {
                title: "Role",
                icon: "isnotblank",
                badge: "new",
                template: function (itemData, itemIndex, element) {
                    let radioGroupDiv = $("<div style='padding:15px'>")
                    radioGroupDiv.appendTo(element);
                    radioGroupDiv.dxRadioGroup({
                        items: ["Owner", "Administrator", "Manager"],
                        value: "Owner"
                    }).dxRadioGroup('instance');
                }

>             }]

Reviewer: This code doesn't relate to the topic.
